plugins/nodes/src/js/pages/nodes/NodeDetailPage.js

```diff
@@ -23,12 +23,12 @@ class NodeDetailPage extends mixin(TabsMixin, StoreMixin) {
     super(...arguments);
 
     this.store_listeners = [
-      { name: "summary", events: ["success"], suppressUpdate: false },
-      { name: "state", events: ["success"], suppressUpdate: false },
+      { name: "summary", events: ["success"], suppressUpdate: true },
+      { name: "state", events: ["success"], suppressUpdate: true },
```

Comment:
removing those we should make sure we have `on<StoreName>Store<EventName>` functions. I.e. in the component we do have `onStateStoreSuccess` but nothing for summary and nodes

Reply:
I mean, nothing breaks like this, so maybe we don't need that anymore

Reply:
can we please ensure checking the components don't rely on data from the stores please?